docs/search/search_for_content.md

```diff
@@ -3,26 +3,43 @@ description: Search through all content in the system and filter the results by
 ---
 
 # Search for content
+
 You can enter a search keyword and the application checks all the searchable Fields and returns a list of search results that contain the keyword.
 
 ![Basic Search](img/basic_search.png)
 
-You can also select a language to search in.
-The results will contain Content items that are translated into this language.
+Everywhere in the Back Office, completion suggestions are proposed while typing in the search box.
+In addition to clicking in it, you can also focus on the search bar using the keyboard shortcut `Ctrl+/` on Windows/Linux, or `Cmd+/` on macOS.
+If a suggestion is what you're looking for, click on it in the suggestion drop-down, or use `down` and `up` keys to highlight it, then press enter to open it.
+If no suggestion correspond to your search, press `Enter` key or click "View all results".
+
+![Suggestion drop-down](img/suggestion.png)
 
+If several languages are available, you can select a language to search in at the top of the left **Filters** menu.
```

Comment:
@dabrt You're right, it would be safer, as the context was set to "Everywhere in the Back Office" earlier.

```suggestion
On the results page, if several languages are available, you can select a language to search in at the top of the left **Filters** menu.
```